iceoryx_posh/test/integrationtests/test_shm_sigbus_handler.cpp

```diff
@@ -28,7 +28,7 @@ using namespace ::testing;
 
 TEST(ShmCreatorDeathTest, AllocatingTooMuchMemoryLeadsToExitWithSIGBUS)
 {
-    const iox::roudi::ShmNameString TEST_SHM_NAME{"/test_name"};
+    const iox::ShmName_t TEST_SHM_NAME{"/test_name"};
```

Comment:
currently it's not user facing, but shall we also remove the leading slash from the shm name and just prepend it in the `SharedMemory` class?

Reply:
Good question. I don't see any reasons besides weak consistency feelings

Reply:
Do it! Or as would Arnold Schwarzenegger would say: " hasta la vista leading slash"

Reply:
Okay. I will create an issue for this. I think this is prio 3 since it doesn't has an impact on our API.

Reply:
@elBoberido can you assign it to me? I'm already trained in getting rid of slashes.

Reply:
done